### PR TITLE
Fill 'preview' field in daemon's `get_version`

### PR DESCRIPTION
**Pull request by benjamreis**
See: https://github.com/xapi-project/xen-api/pull/5047

### CODE REVIEW

**Review by gthvn1 — APPROVED** [review]

File: bin/v6_server.ml [hunk 1]
```diff
@@ -88,4 +88,4 @@ let apply_edition dbg edition params =
 
 let get_editions dbg = editions
 
-let get_version dbg = Version.version
+let get_version dbg = (Version.version, "final")
```
Comment (psafont) [inline bin/v6_server.ml]: This works like a key-value list now, This should give you flexibility on the attributes you want to pass around:
```suggestion
let get_version dbg = [("dbv", Version.version)]
```
or maybe:
```suggestion
let get_version dbg = [
    ("dbv", Version.version)
  ; ("is_final", "true")
]
```

As I said, it's up to xcp-ng on what it wants to offer here. Changing the previously-hardcoded-in-xapi name of "dbv" to something that makes sense outside of citrix, like "version" may be worthwhile but I'd recommend some testing to ensure nothing breaks

**Review by stormi — APPROVED** [review]
LGTM, including always returning is_final = True. We won't really use this value anyway and I don't want to change it between pre-releases and final releases.

**Comment by psafont** [comment]
My opinion is, if you're not going to use it ever, why include it? Xenserver will do its own thing, xcp-ng can do its own

**Comment by stormi** [comment]
I wasn't sure it was optional, but indeed we can also remove it if nothing in XAPI requires it.

**Comment by psafont** [comment]
xapi does not use it to perform any decision. The only think it does with this list is to expose it through the API / xe cli for third parties

**Review by stormi — APPROVED** [review]

**Pull request merged**